master/buildbot/test/unit/test_stats_service.py

```diff
@@ -154,9 +164,9 @@ def test_influx_service_not_inited(self):
         svc.thd_postStatsValue("test", "test", "test")
         self.assertLogged("Service.*not initialized")
 
-    def test_storage_backend_base(self):
-        # Just initlize and run thd_postStatsValue
-        svc = StatsStorageBase()
+    def test_storage_backend_base_failure_on_init(self):
+        svc = DummyStatsStorageBase()
```

Comment:
Yes, of course. I agree too. In the last commit, I subclassed the base ABC and overrode the abstract method. That worked just fine. In this new commit, I subclassed it again and *didn't* override the abstract method because that's was @sa2ajj as far as I could tell. Hence the error.